### PR TITLE
Implemented Playlist management functionality. Kindly review it and let me know what do you think about it.

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**

### CODE REVIEW

File: src/Pages/PlayList/PlayList.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, {useEffect, useState} from 'react';
+import './PlayList.css';
+import { Image, PlayListModal, VideoCard } from "../../Shared"; 
+import { useParams } from 'react-router-dom';
+import { useAlert, usePlaylist } from '../../Context';
+import { getVideos , getplaylists, getPlaylistById} from '../../Helpers/Services/actions';
+import { v4 as uuid } from "uuid";
+import { RiPlayListAddFill } from "react-icons/ri";
+import { MdPlaylistAdd , MdOutlinePlaylistPlay, MdDeleteForever} from "react-icons/md";
+
```
Comment (yaswanthmyneni) [inline src/Pages/PlayList/PlayList.jsx:10]: suggestion: export all icons from index.js and import from index.js file

File: src/Pages/PlayList/PlayList.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                    {/* <!-- LEFT CONTAINER - videos in the playlist... --> */}
+                    <div className="col-9 left-container">                        
+
+                        <Image src="https://www.netmeds.com/images/cms/aw_rbslider/slides/1643642689_Cart_Bannersip.jpg" alt="banner"/>
+                        <div className="col-12 flex-container-row playlist-videos">           
+                            
+                            { playlistVideos && playlistVideos.length && selected_playlist?
+                                playlistVideos.map((cVal,cIndx)=>
+                                <VideoCard key={cIndx} video_data={cVal} module="playlist" selected_playlist={selected_playlist}/>
+                                )
```
Comment (yaswanthmyneni) [inline src/Pages/PlayList/PlayList.jsx:33]: suggestion: use better names.

File: src/Pages/Video/Video.jsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                        <div className="col-12 video-right-container">
+                           {/* will show category videos here */}
+                           {/* functionality yet to be implemented  */}
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+            {/* <PlayListModal/> */}
+        </>
```
Comment (yaswanthmyneni) [inline src/Pages/Video/Video.jsx:79]: changes: remove comments

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes.

File: src/Pages/PlayList/PlayList.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.div-playlist{
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    background-color: var(--lightgray-color);
+    border-radius: 5px;
+    margin: 10px 0;
+    /* height: 2rem; */
```
Comment (thenorthkun) [inline src/Pages/PlayList/PlayList.css]: You can remove commented code

File: src/Pages/Dashboard/RouteData.jsx [hunk 5]
```diff
@@ -23,9 +26,9 @@ const RouteData = () =>{
         <Routes>  
            <Route exact path='/user' element={<PrivateRoute/>}>                
               <Route exact path="/user/liked-videos"  element={<LikedVideosPage/>}/> 
-                {/* <Route exact path='/user/wishlist' element={<Wishlist/>}/> */}
+              <Route exact path='/user/playlists' element={<PlayList/>}/>
             </Route>
-            {/* <Route exact path="/liked-videos"  element={<LikedVideosPage/>}/>  */}
+            <Route exact path="/video/:videoId"  element={<Video/>}/> 
             <Route exact path="/videos/:categoryId"  element={<VideoListingPage/>}/> 
             {/* <Route exact path='/category' element={<VideoListingPage/>}/>  */}
```
Comment (thenorthkun) [inline src/Pages/Dashboard/RouteData.jsx]: You can remove this piece of code

**Review by thenorthkun — APPROVED** [review]
Bug report : create new playlist button shows error 
Everything else looks fine.

File: src/Pages/Dashboard/RouteData.jsx (hunk 5 above)
Comment (kgajriaAgarwal) [reply src/Pages/Dashboard/RouteData.jsx]: ok!

File: src/Pages/PlayList/PlayList.css (hunk 4 above)
Comment (kgajriaAgarwal) [reply src/Pages/PlayList/PlayList.css]: ok

File: src/Pages/Video/Video.jsx (hunk 3 above)
Comment (kgajriaAgarwal) [reply src/Pages/Video/Video.jsx:79]: ok!

File: src/Pages/PlayList/PlayList.jsx (hunk 2 above)
Comment (kgajriaAgarwal) [reply src/Pages/PlayList/PlayList.jsx:33]: ok

File: src/Pages/PlayList/PlayList.jsx (hunk 1 above)
Comment (kgajriaAgarwal) [reply src/Pages/PlayList/PlayList.jsx:10]: ok

**Pull request merged**